source/administration/statistics.rst

```diff
@@ -0,0 +1,66 @@
+Statistics
+================
+
+Statistics on users, posts and channels are tracked for each system and team. Enterprise Editions have access to advanced system statistics.
+
+Site statistics
+-----------------
+
+System statistics are viewable under **System Console** > **Reporting** > **Site Statistics**. The data shown here is a cumulative sum
+across all teams on the system.
+
+**Total Users:** The total number of accounts created on your system, including both active and inactive accounts.
```

Comment:
If "inactive" means DeleteAt > 0 then this description is incorrect. The System Console shows the number of users on the system who's accounts have not been deleted.